sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/notifications/fcm/PushNotificationClient.java

```diff
@@ -71,7 +71,9 @@ public boolean hasStarted() {
      * @param deviceRegistrationToken Device registration token obtained from the FCM SDK.
      * @param errorHandler error handler callback for registration failures
      * @throws RuntimeException if push notifications failed to start.
+     * @deprecated The function will be replaced by startPushNotifications(String deviceRegistrationToken)
```

Comment:
Same comment about using JavaDoc linking, although we probably don't need it for classes in the `implementation` package.